src/test/java/com/artipie/api/artifactory/AddUpdatePermissionSliceTest.java

```diff
@@ -105,16 +107,33 @@ void updatesPermissions() throws IOException {
             this.permissionsForUser(repo, "john"),
             Matchers.containsInAnyOrder("*")
         );
+        MatcherAssert.assertThat(
+            "Sets patterns",
+            this.patterns(repo),
+            Matchers.contains("**")
```

Comment:
@olegmoz I'd suggest to add some more patterns to test json to be sure all of them are stored correctly